api/tasks/alert.go

```diff
@@ -0,0 +1,71 @@
+package tasks
+
+import (
+	"bytes"
+	"html/template"
+	"strconv"
+
+	"github.com/ansible-semaphore/semaphore/models"
+	"github.com/ansible-semaphore/semaphore/util"
+)
+
+const emailTemplate = `Subject: Task '{{ .Alias }}' failed
+
+Task {{ .TaskId }} with template '{{ .Alias }}' was failed!
+Task log: <a href='{{ .TaskUrl }}'>{{ .TaskUrl }}</a>`
+
+type Alert struct {
+	TaskId  string
+	Alias   string
+	TaskUrl string
+}
+
+func (t *task) sendMailAlert() {
+
```

Comment:
this code block (not specific to Go) could be rewritten to be more readable

Example:

```
if true {
  do something
} else {
  do the other stuff
}
```

Could be rewritten to be more expressive as:

```
if false {
  do something
  return
}

do the other stuff
```